config/scheduler-chromeos.yaml

```diff
@@ -389,3 +389,73 @@ scheduler:
     <<: *test-job-arm64-mediatek
     platforms:
       - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-av1-chromium-10bit
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-h264
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8183-kukui-jacuzzi-juniper-sku16
+      - mt8186-corsola-steelix-sku131072
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-h264-frext
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8183-kukui-jacuzzi-juniper-sku16
+      - mt8186-corsola-steelix-sku131072
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-h265
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-vp8
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8186-corsola-steelix-sku131072
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-vp9
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8186-corsola-steelix-sku131072
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-h264
+    <<: *test-job-arm64-qualcomm
+    platforms:
+      - sc7180-trogdor-kingoftown
+      - sc7180-trogdor-lazor-limozeen
```

Comment:
```suggestion
```

This is already included in `test-job-arm64-qualcomm` (same for the following entries)

Reply:
Done